### DISCUSSION TITLE
Example with Supabase

### DISCUSSION THREAD

**Discussion by riderx** [posva/pinia-colada#82] [Ideas]
I think it could be nice to have a example with Supabase, as it's widely used as PaaS

**Comment by posva** [comment]
Examples welcome!
The docs will probably focus on explaining that the query function should return a promise of the data. Then it’s up to whatever layer for fetching you use!

**Comment by Alexbrazdasilva** [comment]
I don't know if this is the case, but using the supabaseClient composable, I only needed to do this arrangement.

```js
export const useLogOutMutation = defineMutation(() => {
  const client = useSupabaseClient();

  return useMutation({
    key: ["logout"],
    mutation: () => client.auth.signOut(),
  });
});

```

**Comment by g1ronn1mo** [comment]
Hi, is there a benefit of using both? For me it seems like there is only a layer of complexity more with no benefit.
